crud/src/main/java/com/redhat/lightblue/crud/LightblueHealth.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.lightblue.crud;
+
+/**
+ *
+ */
+public class LightblueHealth {
+	private final boolean isHealthy;
+	private final String details;
+
+	/**
+	 * @param isHealthy
+	 * @param details
+	 */
+	public LightblueHealth(boolean isHealthy, String details) {
+		this.isHealthy = isHealthy;
+		this.details = details;
+	}
+
+	/**
+	 * @return if the application is healthy
+	 */
```

Comment:
Return doc is not really helpful (no useful information), and in fact is kind of misleading–the health check isn't for an entire application. It's for a specific controller. So, either this should include useful information–like perhaps some guidelines for what "healthy" means about a controller–or we should remove the javadoc.